storybook/cypress/e2e/header.cy.ts

```diff
@@ -287,6 +304,20 @@ describe("Header", () => {
         url: "#profileUrl",
         menuItem: undefined,
         type: "profile",
+      })
+      // Help
+      .get("@dsoHeaderShadow")
+      .find(".dso-header-session .help > a")
+      .click()
+      .get("@headerListener")
+      .invoke("getCalls")
+      .invoke("at", -1)
+      .its("args.0.detail")
```

Comment:
Deze 3 kun je combineren:

```suggestion
      .its("lastCall.args.0.detail")
```

Reply:
Mooi, dan pas ik dat ook aan in de andere plekken waar dit 'patroon' voorkomt in dit testbestand!